lib/git-shell-out-strategy.js

```diff
@@ -656,6 +656,23 @@ export default class GitShellOutStrategy {
     return rawDiffs;
   }
 
+  async getStagedChangesPatch() {
+    const output = await this.exec([
+      'diff', '--staged', '--no-prefix', '--no-ext-diff', '--no-renames', '--diff-filter=u',
```

Comment:
another question just for my own education: how did you know what arguments to pass to `diff` here?

Reply:
great, thanks! that's really helpful.